docs/user-tutorial/benchmarks/micro-benchmarks.md

```diff
@@ -394,6 +394,23 @@ Test the performance of large scale matmul operation with multiple GPUs:
 | pytorch-sharding-matmul/allreduce_time | time (ms) | Time of sharding matmul using allreduce. |
 | pytorch-sharding-matmul/allgather_time | time (ms) | Time of sharding matmul using allgather. |
 
+####
+
+### `dist-inference-simulation`
+
+#### Introduction
+
+Test the performance of a simulated distributed model inference.
```

Comment:
Test the performance of distributed model inference.

Reply:
Fixed